### ISSUE TITLE
Dijkstra v2: animation[0].object is undefined

### ISSUE THREAD

**Issue by atilante** [Aalto-LeTech/jsav-exercise-recorder#87]
Context: test bench, DijkstraPE-research-v2
JAAL 2.0 Validation error:
```
Data does not conform to JAAL 2.0rc1. jsav-exercise-recorder-bundle.js:11049:17
Array [ {…}, {…} ] jsav-exercise-recorder-bundle.js:11050:17
```
Cause:

Dijkstra v2: animation[0].object is undefined

**Comment by atilante** [comment]
When an edge is clicked:

function addStepToSubmission(eventData, dataStructuresState, svgImage) :

line 132:
newState.object = getClickedObject(dataStructuresState);
returns undefined

**Comment by atilante** [comment]
Guess: function getClickedObject(dsState): if this is the first gradable step, one has to compare the current state to the initial state, not the state at the previous animation step (because that does not exist at the first animation step).

**Comment by atilante** [comment]
Only the style change from `unselected` to `selected` is propagated to JAAL, not style change from `unselected` to `queued`.

**Comment by atilante** [comment]
Also: dataStructures/graph/graph.js:    style: classes.contains("marked") ? "selected" : "unselected",

**Comment by atilante** [comment]
Fixed in 6b1a151
